.gitignore

```diff
@@ -5,6 +5,7 @@ dist
 .vscode/
 hack/kind.test.yaml
 coverage.txt
+test.data
```

Comment:
this was being added during the new release, see error in this action:
[https://github.com/jonshaffer/kube-bench/runs/3614436216?check_suite_focus=true](https://github.com/jonshaffer/kube-bench/runs/3614436216?check_suite_focus=true)